### ISSUE TITLE
Automatically switch themes based off of the OS theme

### ISSUE THREAD

**Issue by kvhjdffv** [fmhy/edit#2536]
### Type

Wiki suggestion (Quality of life)

### Add additional context

Hello, I was wondering if you could implement something that will:

1. Check whichever theme a user is using
2. Then automatically switch to that theme

Examples:

![2024-10-27 18_08_36-Settings](https://github.com/user-attachments/assets/9df602f6-d064-4208-98f1-db2390ee103e)

Or

![2024-10-27 18_12_19-Settings](https://github.com/user-attachments/assets/5f901b20-3315-4075-a6b4-f078d15de9d0)

In other words, could you make it so that the default theme is "Automatic" instead of "Dark"? Thanks.

**Comment by kvhjdffv** [comment]
Not on Brave :(

**Comment by InternetNinjo** [comment]
> Not on Brave :(

![image](https://github.com/user-attachments/assets/686c0542-1773-4569-bc75-15a51638d2ac)
Not even on chromium at all, because brave also uses chromium as base

**Comment by InternetNinjo** [comment]
> It is already implemented. It works on Firefox, at least.

that doesn't mean it works on others? please test before saying things

**Comment by rizdarmwn** [comment]
Should be okay now on #2541 

I have checked on brave and it's working by changing the "Brave colors" settings on "Appearance"

Please do tell if on yours is not working.

**Comment by InternetNinjo** [comment]
> Should be okay now on #2541
> 
> I have checked on brave and it's working by changing the "Brave colors" settings on "Appearance"
> 
> Please do tell if on yours is not working.

Each browser set to light mode, testing...

Edge: Not Working ⬇️ 
![image](https://github.com/user-attachments/assets/9d4e0f4a-f743-444b-86ac-ff0bd45a9b96)
Chrome: Not Working ⬇️ 
![image](https://github.com/user-attachments/assets/9601c278-5b8f-4803-95f3-6548397710b9)
Brave: Not Working ⬇️ 
![image](https://github.com/user-attachments/assets/3b8b0042-80f2-47b0-9efb-a40665dc9217)

I've tested it previously with Chromium which is what all these three browsers are based on.

**Comment by rizdarmwn** [comment]
It's not deployed to the net yet @InternetNinjo 

If you can test locally, that would be great. If not, let's wait for it to be merged.

**Comment by InternetNinjo** [comment]
> It's not deployed to the net yet @InternetNinjo
> 
> If you can test locally, that would be great. If not, let's wait for it to be merged.

Alright i understand.

**Comment by kvhjdffv** [comment]
@rizdarmwn @nbats @InternetNinjo Sorry, I fell asleep, Anyway, it still doesn't work?

I have switched to light mode for this btw:
![2024-10-28 18_06_42-Settings](https://github.com/user-attachments/assets/3cd6fff4-bef5-4150-b2e6-ebcd41df1cd1)


Still:
![2024-10-28 18_05_29-](https://github.com/user-attachments/assets/bce47e5a-bcfb-4d76-9f12-96bc87055b93)


:(((

**Comment by kvhjdffv** [comment]
@rizdarmwn @nbats @InternetNinjo Sorry, I fell asleep, Anyway, it still doesn't work?

I have switched to light mode for this btw:
![2024-10-28 18_06_42-Settings](https://github.com/user-attachments/assets/3cd6fff4-bef5-4150-b2e6-ebcd41df1cd1)


Still:
![2024-10-28 18_05_29-](https://github.com/user-attachments/assets/d192840c-8bdd-46c1-be73-d3af1c01d343)

:(((

**Comment by rizdarmwn** [comment]
[testing_lightmode.webm](https://github.com/user-attachments/assets/3984bef6-e1cf-4e93-8b2a-dd0b030115ab)

**Comment by rizdarmwn** [comment]
Should be okay now @kvhjdffv :)

**Comment by kvhjdffv** [comment]
Yes, it works!

Light mode:

![2024-10-28 18_57_25-Settings](https://github.com/user-attachments/assets/0fa7a1f9-5a93-440c-b11c-83d58183151b)

Dark mode:

![2024-10-28 18_58_21-Window](https://github.com/user-attachments/assets/1c0d5cd9-0441-40e0-a8de-13f3641c8ab0)

Thanks :)))
